library/block/block_extensions/src/main/java/org/quiltmc/qsl/block/extensions/api/QuiltBlockSettings.java

```diff
@@ -265,6 +265,31 @@ public QuiltBlockSettings collidable(boolean collidable) {
 		return this;
 	}
 
+	public QuiltBlockSettings opaque(boolean opaque) {
+		((AbstractBlockSettingsAccessor) this).setOpaque(opaque);
+		return this;
+	}
+
+	public QuiltBlockSettings ticksRandomly(boolean ticksRandomly) {
+		((AbstractBlockSettingsAccessor) this).setRandomTicks(ticksRandomly);
+		return this;
+	}
+
+	public QuiltBlockSettings dynamicBounds(boolean dynamicBounds) {
+		((AbstractBlockSettingsAccessor) this).setDynamicBounds(dynamicBounds);
+		return this;
+	}
+
+	public QuiltBlockSettings requiresTool(boolean requiresTool) {
+		((AbstractBlockSettingsAccessor) this).setToolRequired(requiresTool);
+		return this;
+	}
+
+	public QuiltBlockSettings isAir(boolean isAir) {
```

Comment:
I really don't like using a getter-like name for this.
Maybe `airLike(boolean isAir)`?

Reply:
Ah, I thought the normal setter was also `settings.isAir()`, but apparently it's `settings.air()`

Reply:
```suggestion
	public QuiltBlockSettings air(boolean isAir) {
```

Just change that and I'll be good with FCP-ing this PR.